core/standalone/src/main/scala/org/apache/openwhisk/standalone/StandaloneOpenWhisk.scala

```diff
@@ -96,13 +95,37 @@ class Conf(arguments: Seq[String]) extends ScallopConf(arguments) {
 
   val userEvents = opt[Boolean](descr = "Enable User Events along with Prometheus and Grafana", noshort = true)
 
+  val all = opt[Boolean](
+    descr = "Enables all the optional services supported by Standalone OpenWhisk like CouchDB, Kafka etc",
```

Comment:
This is great, should we update `README` too? 
```
      --kafka-ui                   Enable Kafka UI

      --all                        Enables all the optional services supported
                                   by Standalone OpenWhisk like CouchDB, Kafka
                                   etc
```

Reply:
Updated the help docs